src/llvm/LLVMDG2Dot.h

```diff
@@ -6,6 +6,10 @@
 #include "DG2Dot.h"
 #include "llvm/LLVMNode.h"
 
+#if ((LLVM_VERSION_MAJOR == 3) && (LLVM_VERSION_MINOR <= 6))
```

Comment:
I don't test with LLVM 3.6, but this breaks the build for LLVM 3.5 and I'd expect for 3.6 instead.

[DebugInfo.h was moved between 3.4 and 3.5](https://github.com/llvm-mirror/llvm/commit/f4ec8bfaecef4e38f713b9e05d89869b023e1ce8)

Bit more clear is looking at the relevant directory in the release_35 branch:

https://github.com/llvm-mirror/llvm/tree/release_35/include/llvm

Curiously the "DebugInfo.h" header isn't needed for me on LLVM 3.4, 3.5, 3.7 ... That said I think `llvm/IR/DebugInfo.h" is an appropriate path for all LLVM versions since 3.4.

I'll test to confirm and get back to you with a PR...